nucliadb/nucliadb/search/predict.py

```diff
@@ -262,7 +262,7 @@ async def rephrase_query(self, kbid: str, item: RephraseModel) -> str:
             headers=self.get_predict_headers(kbid),
         )
         await self.check_response(resp, expected=200)
-        return await resp.text()
+        return await resp.json()
```

Comment:
Isn't this going to return a dict? This changes the types expected.

Reply:
The problem is that learning is returning a json response, but the payload is simply a string (which is json compatible, so there is nothing wrong...)

But the fact that we were parsing it as text, it made the string be wrapped in quotes...